### DISCUSSION TITLE
Is the CodeGeeX2 JetBrains IDE plugin open source?

### DISCUSSION THREAD

**Discussion by coolbeevip** [THUDM/CodeGeeX2#223] [Q&A] [answered]
Is the CodeGeeX2 JetBrains IDE plugin open source?

**Comment by crazygo** [comment, marked as answer]
Thanks for reaching out, it's not open-source at the moment

**Reply by coolbeevip** [reply]
What user data will the CodeGeeX2 JetBrains IDE plugin collect? Will it collect the project's git address?
